### ISSUE TITLE
mailing list instead of cc-ed emails

### ISSUE THREAD

**Issue by rapidlasso** [ASPRSorg/LAS#10]
Currently we still exchange emails via being cc-ed on personal emails. Can we switch to a mailing list such as google groups or else? I know that Jason Stoker had created some group. Has there been any activity?  I have not checked it as it seems to be behind the ASPRS paywall and thereby not providing the community transparency that I was hoping for ...

**Comment by jstoker74** [comment]
Martin, I spoke with ASPRS leadership at the ITGF and they agreed it would
be good to make those communities visible to more people than just ASPRS
members. I am not sure how long that will take to make those changes
though.  There has not been any activity on there other than myself talking
to myself on there currently.

Jason

**Comment by esilvia** [comment]
I think this forum will work pretty well for public discussions re: clarification and modification of the specification itself. It exists as a permanent record, is free to anyone wanting to join, supports on-demand email subscriptions (by watching the repository and/or individual issues), and permits input from the community outside the LWG.

Jason's suggestion of the ASPRS web forum could be a good location for general LAS discussion, interpretation, education, and outreach, if they're able to lift the "paywall" issue. I agree that this is a significant barrier. Additionally, I've reached out to Jason concerning the need for a means to "subscribe" to the channel and receive notifications about changes.

I'm considering creating a Google Group for conducting administrative LWG business (membership, scheduling, discipline), but haven't taken the dive into that yet. Any input on what else it would be useful for?

**Comment by hobu** [comment]
* There should be a low-volume mailing list for announcements, calls for participation, votes (if any), and document releases. A Google Group would be perfect.

* We need a pointer from the ASPRS LAS page to the github workspace. The workspace is not really findable via google at the moment.

**Comment by esilvia** [comment]
I can create a Google Group using my personal account, unless there's some official ASPRS mechanism for doing this. Is this Google Group the same as the LWG, or is it different?

Along that vein, is there still a difference between participating here and being a member in the LWG?

**Comment by rapidlasso** [comment]
It would be important to assure that the Google Groups account cannot be deleted by a single member or my the chair of the group. Prior to me joining the LWG there was a google group called "[ASPRS-LAS](http://groups.google.com/group/ASPRS-LAS)" being composed of the LWG members. But the LWG chair at the time seems to have simply deleted this forum when its open and publicly available messages became inconvenient during the great LAS 1.4 debate of 2011. While I do not anticipate this to repeat I would still suggest that any new message board will be set up such that its deletion requires more than just one person's permission. Maybe having multiple administrators assures this?

**Comment by esilvia** [comment]
I believe that for Google Groups only the owner can delete a group, and I don't know of a way enforce an approval requirement by the managers of the group. Is there another tool that might work better? The [GitHub repository itself also has this problem](https://stackoverflow.com/q/48215184/1666676).

**Comment by esilvia** [comment]
CC'd email conversations have been almost nonexistent for the past two years, and in every instance I have been able to redirect them to the GitHub Issues pages.

I also added the LWG membership [as a page in the wiki](https://github.com/ASPRSorg/LAS/wiki/LAS-Working-Group-Membership) if anyone wants to send an email blast. I use this list and the responses to the [signup Google Form](https://goo.gl/forms/ltoX0XjiQJyy0YxK2) to generate email lists for reminders about meetings.

I'm going to close this thread since the current systems appear to be working.
